docs/blockchain-rpc/hedera-rpc.md

```diff
@@ -0,0 +1,177 @@
+---
+description: Hedera JSON-RPC Methods
+---
+
+# Hedera
+
+The following JSON-RPC methods are for native integration into Hedera with the [Hedera™ Hashgraph JavaScript SDK](https://github.com/hashgraph/hedera-sdk-js). Hedera also has an open-source project implementing the [Ethereum JSON-RPC standard](https://docs.hedera.com/hedera/core-concepts/smart-contracts/json-rpc-relay) which is not covered in this documentation.
+
+## Methods
+
+- [`hedera_signAndExecuteTransaction`](#hedera_signandexecutetransaction)
+- [`hedera_signAndReturnTransaction`](#hedera_signandreturntransaction)
+- [`hedera_signMessage`](#hedera_signmessage)
+
+## hedera_signAndExecuteTransaction
+
+The `hedera_signAndExecuteTransaction` method is a generic method for sending a transaction that has been converted to bytes to the wallet for signing and execution. The dApp can therefore construct any type of Hedera transaction with the [`hedera-sdk-js` library](https://github.com/hashgraph/hedera-sdk-js)
+
+### Parameters
+
+    1. `Object` - signAndExecuteTransaction parameters
+      1.1 `transaction` : `Object` - transaction parameters
+        1.1.1 `type` : `String` - The Hedera function name (*1)
+        1.1.2 `bytes` : `String` - Transaction that has been converted to bytes array (*2) and encoded as a base64 string.
+
+\*1. The full list of functions can be found [here](https://hashgraph.github.io/hedera-protobufs/#proto.HederaFunctionality), and are available via the [`RequestType` class](https://github.com/hashgraph/hedera-sdk-js/blob/develop/src/RequestType.js) from the Hedera SDK.
+
+\*2. Transactions generated with the Hedera SDK can be converted to bytes with the [`toBytes` method](https://github.com/hashgraph/hedera-sdk-js/blob/50d7f76ffd1fd797a029192fab08b035720998b9/src/transaction/Transaction.js#L1015).
+
+### Returns
+
+    1. `Object` - Result of transaction submission to Hedera network
+      1.1. `response` : `Object` - Response from Hedera network from transaction submission
+        1.1.1. `nodeId` : `String` - The Hedera node the transaction was submitted to
+        1.1.2. `transactionHash` : `String` - The hash of the transaction
+        1.1.3. `transactionId` : `String` - Tranasction ID, which included the payer Account ID and the consensus timestamp
```

Comment:
TransactionIDs famously have different formats in different places with combinations of "@.-", should we specify what the format is here ? Maybe it's `transactionId.toString()` from the SDK ?

Reply:
yep, we should be very specific with this. in fact looks like this description is wrong, as it references the consensus timestamp vs. the valid start timestamp

we can put a link to https://docs.hedera.com/hedera/sdks-and-apis/sdks/transactions/transaction-id